docs/user-guide/configuration.md

```diff
@@ -613,16 +613,17 @@ themes do).
 
 There are two options for prebuilding the index:
 
-Using [Node.js] setting `prebuild_index` to `True` or `node`. This option
+Using [Node.js] by setting `prebuild_index` to `True` or `node`. This option
 requires that Node.js be installed and the command `node` be on the system
 path. If this feature is enabled and fails for any reason, a warning is issued.
 You may use the `--strict` flag when building to cause such a failure to raise
 an error instead.
 
-Using [Lunr.py] setting `prebuild_index` to `python`. Lunr.py is installed
-as part of mkdocs and guarantees compatibility with Lunr.js even on languages
-other than English. If you find substantial inconsistencies or problems please
-report it on [Lunr.py's issues] and fall back to the Node.js version.
+Using [Lunr.py] by setting `prebuild_index` to `python`. This option requires
+that Lunr.py is installed in your Python environment (`pip install lunr`). For
```

Comment:
I originally had that. But changed it.  The issue is that for anyone using anything other than English, they also need `lunr[languages]` and I didn't create an entry for that (what would I even name it?). To make the documentation consistent, I including instructions directing the user to install the libs directly rather than as sub-packages of MkDocs. Maybe we should not including the entry in `extras_require` at all.

Reply:
I would opt for keeping it consistent within the use of lunr.py, i.e. if you need lunr.py you need to `pip install lunr`, if you also need language support, then `pip install lunr[languages]`. It feels clearer to me and if the user has any problems they can direct them to the lunr.py repo.